compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java

```diff
@@ -12018,13 +12030,11 @@ private STNode parseOrderKey(boolean isRhsExpr) {
      *
      * @return Parsed node
      */
-    private STNode parseSelectClause(boolean isRhsExpr) {
+    private STNode parseSelectClause(boolean isRhsExpr, boolean allowActions) {
         startContext(ParserRuleContext.SELECT_CLAUSE);
         STNode selectKeyword = parseSelectKeyword();
 
-        // allow-actions flag is always false, since there will not be any actions
```

Comment:
Shall we also get rid of the preceding newline where this comment was there?
Now keeping a newline does not make sense.